### PR TITLE
Bump default docker tags in hosting webknossos==24.06.0, fossildb==master__489 for latest release

### PR DESCRIPTION
**Pull request by aaronkanzer**
Just a PR for clarity when deploying WebKNOSSOS locally based off of last released changelog https://github.com/scalableminds/webknossos/blob/7b95132dfa46d8c6a5a07e90830bf6144291918e/CHANGELOG.released.md?plain=1#L10

I noticed the discrepancy when I had initially installed 24.04.0, then bumped the version in Docker with a forked version of 24.06.0, thus seeing discrepancies.

Also, bumped compatible fossildb tag as well

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- abc

### TODOs:
- [ ] ...

### Issues:
- fixes #

------
(Please delete unneeded items, merge only when none are left open)
- [ ] Updated [changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [ ] Updated [migration guide](../blob/master/MIGRATIONS.unreleased.md#unreleased) if applicable
- [ ] Updated [documentation](../blob/master/docs) if applicable
- [ ] Adapted [wk-libs python client](https://github.com/scalableminds/webknossos-libs/tree/master/webknossos/webknossos/client) if relevant API parts change
- [ ] Removed dev-only changes like prints and application.conf edits
- [ ] Considered [common edge cases](../blob/master/.github/common_edge_cases.md)
- [ ] Needs datastore update after deployment

### CODE REVIEW

**Comment by aaronkanzer** [comment]
Also -- I signed https://github.com/scalableminds/webknossos/pull/7881#issuecomment-2166692706 but it did not register correctly apparently (under my email `aaronkanzer@gmail.com`)

**Comment by MichaelBuessemeyer** [comment]
Hi Aaron, thanks a lot for your contribution. 

To merge your changes into our master, we need you to sign the CLA properly. Apparently, it is not working because you have not added the email address with which you configured your local git to "sign" commits. 
This can be seen by your icon not appearing left to the commit message in this PR (compared to Norman's commit just now)

You can get this email address by executing `git config user.email` in your CLI. Then please add this email address to your GitHub account. A how-to is linked there (I think)

> Aaron Kanzer seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

Hopefully, everything should work then.

File: tools/hosting/docker-compose.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ services:
 
   # FossilDB
   fossildb:
-    image: scalableminds/fossildb:master__442
+    image: scalableminds/fossildb:master__489
```
Comment (aaronkanzer) [inline tools/hosting/docker-compose.yml:61]: Cc @normanrz -- this resolved https://forum.image.sc/t/ambiguity-for-setting-application-conf-for-serving-annotations/98112/5

thank you again!

**Comment by fm3** [comment]
Closing this PR since its contents are now supserseded by https://github.com/scalableminds/webknossos/pull/8287. Still, thanks for your contribution and sorry about the CLA hassle

**Pull request closed without merging**